BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/CaptureFragment.java

```diff
@@ -0,0 +1,51 @@
+package com.blueprint.foe.beetracker;
+
+import android.app.Fragment;
```

Comment:
I see you use appcompatactivity so you should probably use a supporting fragment class to support.